### ISSUE TITLE
Domain Name

### ISSUE THREAD

**Issue by abhurtun** [jschniper/gelf_logger#3]
Pass in a domain name rather than an ip address for the host config parameter
host: "garylog.com",

**Comment by jschniper** [comment]
I've just pushed some changes to the develop branch. Do you mind giving them a test run? If everything works as expected, I'll go ahead and create a new release.

**Comment by jschniper** [comment]
@abhurtun, I just wanted to check in with you and see if you had given the new code a try

**Comment by abhurtun** [comment]
Sorry was away will try it and get back to you

**Comment by sstine** [comment]
I just tested this with v0.2.1 and it is only working with an IP address, not a hostname. Providing a hostname causes a crash:

```
** (Mix) Could not start application logger: Logger.App.start(:normal, []) returned an error: shutdown: failed to start child: Logger.Watcher
```

**Comment by mcoms** [comment]
Hey @jschniper, thanks for this — your new code seems to work great.

```
defp deps do
  [{:gelf_logger, "~> 0.2.0", git: "https://github.com/jschniper/gelf_logger.git", branch: "develop"} ... ]
end

config :logger, :gelf_logger,
  host: "graylog.example.com", ...
```

Verified with Wireshark. I tested a FQDN and an IPv4 address. Both work. Thanks!

@sstine, this patch isn't in v0.2.1/released yet.

**Comment by jschniper** [comment]
I've released a 0.3.0 version of the package. @mcoms, thank you for giving it a try.
